src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -316,7 +288,156 @@ To ensure your CSS files are accessible, add the `crossorigin="anonymous"` attri
 Here's an example of adding the `crossorigin="anonymous"` attribute:
 
 ```
- <link rel="stylesheet" href="assets.yoursite.com/styles.css" crossorigin="anonymous">
+<link rel="stylesheet" href="assets.yoursite.com/styles.css" crossorigin="anonymous">
 ```
 </Collapser>
 </CollapserGroup>
+
+## Manually record session replays [#manual-replays]
+
+If needed, you can programmatically invoke a session replay using these two API methods:
+
+* [`NREM.startSessionReplay`](/docs/browser/new-relic-browser/browser-apis/start/)
+* [`NREM.stopSessionReplay`](/docs/browser/new-relic-browser/browser-apis/stop/)
+
+Here are some use cases:
+
+* Allow users to opt in for interaction recording. For example, if you have a pop-up asking "Allow this session to be recorded for performance analysis?":
+    1. Upon user consent, call `NREM.startSessionReplay` to initiate recording. 
+    2. Once the session is complete or consent is withdrawn, use `NREM.stopSessionReplay` to stop capturing data.
+* Record sessions based on specific criteria. For example:
+    * Start recording only for sessions on specific URLs such as a product checkout flow.
+    * Stop recording for sensitive pages such as `/billing` or `/health-records`.
+
+To ensure all user interactions are captured during manual recording, set sampling rates to 0% in your application settings.
+
+## Use Nerdgraph to configure replay settings [#nerdgraph-mutation]
```

Comment:
I kind of want to move this info to the Nerdgraph section of the docs.

Reply:
I think I'll include this in a post-release PR. Good idea.